.circleci/config.yml

```diff
@@ -0,0 +1,130 @@
+version: 2.1
+orbs:
+  ci: bigcommerce/internal@volatile
+  node: bigcommerce/internal-node@volatile
+
+#####################################################
+# Aliases
+#####################################################
+aliases:
+  - &node_executor
+      executor:
+        name: node/node
+        node-version: "10"
+
+  - &release_filter
+      branches:
+        only:
+          - master
+
+  - &pull_request_filter
+      branches:
+        ignore:
+          - master
+
+#####################################################
+# Commands
+#####################################################
+commands:
+  install:
+    steps:
+      - ci/pre-setup
+      - restore_cache:
+          keys:
+            - bigpay-client-js
+      - run: npm ci
+      - save_cache:
+          key: bigpay-client-js
+          paths:
+            - ~/.npm
+  test:
+    steps:
+      - run: npm run prerelease
```

Comment:
I don't think `prerelease` is a separate command. It is intended to be called automatically by NPM when you run `release` command. So you should either run `npm run lint && npm test` here or define a new alias command for it.

Reply:
fixed